source/connect.txt

```diff
@@ -105,6 +105,45 @@ deployment from which you were disconnected:
 
 .. include:: /includes/steps/activate-connection.rst
 
+Launch MongoDB Shell
+~~~~~~~~~~~~~~~~~~~~
+
+You can connect the
+`MongoDB Shell <https://docs.mongodb.com/mongodb-shell/>`__ or legacy
+:binary:`mongo <mongo>` shell to your active deployment.
+
+Considerations
+``````````````
+
+- The shell that |vsce| uses to connect to your deployment is
+  determined by the :guilabel:`Shell` setting in your
+  :ref:`extension settings <vsce-settings>`. You can choose either
+  the `MongoDB Shell <https://docs.mongodb.com/mongodb-shell/>`__ or
+  legacy :binary:`mongo <mongo>` shell.
+
+- The path to your desired shell must exist in your system's ``PATH``.
+  If the shell specified in your :ref:`settings <vsce-settings>` does
+  not exist in your ``PATH``, |vsce| attempts to use the other shell. If
+  neither shell exists in your ``PATH``, the operation errors.
```

Comment:
Currently we don't fallback. We just try to use the shell that the user has set in their settings, and if it fails we don't try the other we just fail with a message. We could add that in in future work.
Here's what it looks like when it doesn't exist in path:
<img width="537" alt="Screen Shot 2020-06-25 at 3 16 21 PM" src="https://user-images.githubusercontent.com/1791149/85727053-fcefb780-b6f6-11ea-9b47-84181afbe29f.png">

Maybe we can just remove the part about fallback?

Reply:
Edit* forgot to ping @jeff-allen-mongo

Reply:
@anemy Oh interesting, I thought I'd seen in my testing that it did fall back to the other shell, but now when I'm trying to reproduce it it just fails as you say.

My bad, must've not had things set up the way I thought I did while testing. I'll remove this bit about fallback.